react-components/src/components/Toolbar/Image360HistoricalOverviewToolbar.tsx

```diff
@@ -0,0 +1,110 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { Detail, Flex } from '@cognite/cogs.js';
+import React from 'react';
+import styled from 'styled-components';
+import { Thumbnail } from '../utils/Thumbnail';
+
+export interface Image360RevisionDetails{
+  revisionDate?: string;
+  revisionImageUrl?: string;
+};
+
+export interface Image360HistoricalOverviewToolbarProps{
+  stationId?: string;
+  stationName?: string;
+  collectionId?: string;
+  revisionCollection?: Image360RevisionDetails[];
+};
+
+export const Image360HistoricalOverviewToolbar = ({stationId, stationName, collectionId, revisionCollection}: Image360HistoricalOverviewToolbarProps) => {
+  return(
+    <OverviewContainer>
+
+      <StyledSubFlex direction='column'>
+        <StyledFlex>{stationName}</StyledFlex>
+        <StyledDetail>Station: {stationId}</StyledDetail>
+        <StyledDetail>Collection: {collectionId}</StyledDetail>
+      </StyledSubFlex>
+
+    <StyledLayoutGrid>
```

Comment:
It's fine, I'm just wondering why the `Styled`-prefix is added to some of the variables here? A convention for something?